File: erettsegi_emelt/2022_may/java/Telek.java
public class Telek {

    public final int adoszam;
    public final String utcaNev;
    public final String hazSzam;
    public final String adosav;
    public final int terulet;

    public Telek(String[] data) {
        adoszam = Integer.parseInt(data[0]);
        utcaNev = data[1];
        hazSzam = data[2];
        adosav = data[3];
        terulet = Integer.parseInt(data[4]);
    }
}

File: erettsegi/emelt/2022_may/c#/Telek.cs
public class Telek {

    public readonly int adoszam;
    public readonly string utcaNev;
    public readonly string hazSzam;
    public readonly string adosav;
    public readonly int terulet;

    public Telek(string[] data) {
        adoszam = int.Parse(data[0]);
        utcaNev = data[1];
        hazSzam = data[2];
        adosav = data[3];
        terulet = int.Parse(data[4]);
    }
}